### PR TITLE
Added a section for the Diagnostics feature

### PR DESCRIPTION
**Pull request by gunamata**
- Restructured the features page a bit to show only top 2 levels in the right hand side nav
- Added a section for the Diagnostics feature

Signed-off-by: Gunasekhar Matamalam <gunasekhar.matamalam@suse.com>

### CODE REVIEW

File: docs/getting-started/features.md [hunk 1]
```diff
@@ -100,7 +98,19 @@ To perform a factory reset:
...
 1. Optional: start Rancher Desktop again.
 
-# Preferences
+## Diagnostics
+
+**Note:** Rancher Desktop *doesn't* send the diagnostics data to any remote server for processing or storing.
+
+The **Diagnostics** feature runs several checks in the background to detect common problems such as missing minimum requirements, misconfigurations, etc., in your environment to help you self-troubleshoot and fix Rancher Desktop application issues.
+
+The diagnostics checks are run every time during the application launch. If there are any problems identified then the problems count is shown next to the *Diagnostics* menu text in the left navigation bar, indicating that something needs your attention. The *Diagnostics* tab displays the results of the diagnostics tests highlighting areas that need attention and guides you to resolve problems.
```
Comment (jandubois) [inline docs/getting-started/features.md:107]: I would add some commas, but maybe check with a docs person about that. 😄 

```suggestion
The diagnostics checks are run every time during the application launches. If there are any problems identified then the count of failed checks is shown next to the *Diagnostics* menu text in the left navigation bar, indicating that something needs your attention. The *Diagnostics* tab displays the results of the diagnostics tests, highlighting areas that need attention, and guides you to resolve problems.
```

File: docs/getting-started/features.md [hunk 2]
```diff
@@ -100,7 +98,19 @@ To perform a factory reset:
...
+
+On this tab, you can mute/unmute individual checks that the diagnostics tool performs to fit your needs. You can also rerun the diagnostics anytime within an active Rancher Desktop session.
```
Comment (jandubois) [inline docs/getting-started/features.md:109]:
```suggestion
On this tab you can mute/unmute individual checks if you have a non-standard setup and know that these checks don't apply to your situation. You can also rerun the diagnostics anytime to verify that changes you have made to your environment have rectified the problem.
```

**Comment by jandubois** [comment]
Note that PR is still in "draft", which normally means it is not ready for review yet. 😄 

Also in general I prefer to separate structural changes (the modifications of heading levels throughout the docs) from content changes in a specific area (but don't bother splitting up this PR; just general feedback for the future).

File: docs/getting-started/features.md (hunk 1 above)
Comment (gunamata) [reply docs/getting-started/features.md:107]: The commas do help in reading, especially with the long sentences that I tend to use.. :)

File: docs/getting-started/features.md [hunk 3]
```diff
@@ -100,7 +98,19 @@ To perform a factory reset:
...
 1. Optional: start Rancher Desktop again.
 
-# Preferences
+## Diagnostics
+
+**Note:** Rancher Desktop *doesn't* send the diagnostics data to any remote server for processing or storing.
+
+The **Diagnostics** feature runs several checks in the background to detect common problems such as missing minimum requirements, misconfigurations, etc., in your environment to help you self-troubleshoot and fix Rancher Desktop application issues.
+
+The diagnostics checks are run every time during the application launches. If there are any problems identified then the count of failed checks is shown next to the *Diagnostics* menu text in the left navigation bar, indicating that something needs your attention. The *Diagnostics* tab displays the results of the diagnostics tests, highlighting areas that need attention, and guides you to resolve problems.
```
Comment (jandubois) [inline docs/getting-started/features.md:107]: So sorry I missed this last round:

```suggestion
The diagnostics checks are run every time when the application launches. If there are any problems identified then the count of failed checks is shown next to the *Diagnostics* menu text in the left navigation bar, indicating that something needs your attention. The *Diagnostics* tab displays the results of the diagnostics tests, highlighting areas that need attention, and guides you to resolve problems.
```

**Review by jandubois — APPROVED** [review]
LGTM now, but the image will need updating from the RC build.

**Pull request merged**